.travis.yml

```diff
@@ -51,10 +51,10 @@ env:
   - MAIN_TARGET="world"
   # Main test suites
   matrix:
-  - TEST_TARGET="test-suite" COMPILER="4.02.3+32bit"
+  - TEST_TARGET="test-suite" COMPILER="4.02.3+32bit" EXTRA_CONF="-warn-error yes"
   - TEST_TARGET="validate"                           TW="travis_wait"
   - TEST_TARGET="validate"   COMPILER="4.02.3+32bit" TW="travis_wait"
-  - TEST_TARGET="validate"   COMPILER="${COMPILER_BE}+flambda" CAMLP5_VER="${CAMLP5_VER_BE}" NATIVE_COMP="no" EXTRA_CONF="-flambda-opts -O3" EXTRA_OPAM="num" FINDLIB_VER="${FINDLIB_VER_BE}"
+  - TEST_TARGET="validate"   COMPILER="${COMPILER_BE}+flambda" CAMLP5_VER="${CAMLP5_VER_BE}" NATIVE_COMP="no" EXTRA_CONF="-flambda-opts -O3 -warn-error yes" EXTRA_OPAM="num" FINDLIB_VER="${FINDLIB_VER_BE}"
```

Comment:
I believe you should not add `-warn-error yes` with the bleeding edge compiler. We do not test that. @SkySkimmer should confirm though.

Reply:
I think we do.

Reply:
We do test warnings with newer ocaml.